docs/reference/mobile/android/setup.md

```diff
@@ -2,36 +2,88 @@
 
 # Setting up Android support for the GDK
 
+<%(Callout message="
+Before reading this document, make sure you have read:
+
+  * [Mobile support overview]({{urlRoot}}/reference/mobile/overview)
+  * [Creating workers with WorkerConnector](https://docs.improbable.io/unity/alpha/reference/workflows/monobehaviour/creating-workers)
+")%>
+
 ## Get the dependencies for developing Android games
 
-1. Follow the steps in [Setup & installation]({{urlRoot}}/machine-setup) and be sure to select the **iOS build support** component during your Unity installation.
+1. Follow the steps in [Setup & installation]({{urlRoot}}/machine-setup) and be sure to select the **Android build support** component during your Unity installation.
```

Comment:
Can we assume users have Unity installed if they are here? 

If so - we can direct them to modify the installation using the Hub or otherwise rather than sending them to the full setup guide

Reply:
we can't assume that. they may or may not have it installed

Reply:
Maybe break this down into two options: one if you are a new user entirely and one for I now want to add mobile build targets. Then its simply a case of:

If new user, follow setup guide + make sure to select relevant build support.
If existing user, modify your existing Unity installation

As these really are different things.